test/e2e/common/util.go

```diff
@@ -889,6 +889,66 @@ func IsVolumePublished(uuid string) bool {
 	return true
 }
 
+func CheckNoVolumeResources() (bool, bool) {
+	logf.Log.Info("CheckNoVolumeResources")
+	foundResources := false
+	success := true
+
+	pvcs, err := gTestEnv.KubeInt.CoreV1().PersistentVolumeClaims("default").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Error(err, "CheckNoVolumeResources: list PVCs failed.")
+		success = false
+	} else if pvcs != nil && len(pvcs.Items) != 0 {
+		logf.Log.Info("CheckNoVolumeResources: found PersistentVolumeClaims",
+			"PersistentVolumeClaims", pvcs.Items)
+		foundResources = true
+	}
+
+	pvs, err := gTestEnv.KubeInt.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Error(err, "CheckNoVolumeResources: list PVs failed.")
```

Comment:
You don't set success to false here, is that intentional?

Reply:
oversight on my part, fixed.